backend/user_management/routers/product.py

```
@@ -0,0 +1,140 @@
+
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+from typing import List
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)